Gruntfile.js

```diff
@@ -42,20 +42,7 @@ module.exports = (grunt) => {
       module: {
         rules: [
           {
-            test: /\.js?$/,
-            exclude: /(node_modules|bower_components)/,
-            use: [
-              {
-                loader: 'babel-loader',
-                options: {
-                  presets: [preset],
-                  plugins: ['@babel/plugin-transform-object-assign'],
-                },
-              },
-            ],
-          },
-          {
-            test: /\.ts?$/,
+            test: /\.?s?$/,
```

Comment:
so this will also include css, scss ... do we need such generic regex?

Reply:
Done changed to `test: /(\.ts?$|\.js?$)/`